app/models/comment.rb

```diff
@@ -1,6 +1,6 @@
-class Comment < ApplicationRecord
+class Comments < ApplicationRecord
```

Comment:
Hello, the test is still not running but I have investigated the reasons why and this review will be able to guide you.

- [x] It is a rails convention that model names should be written in the singular form. I can see that you changed `Comment` to `Comments`. Rspec is throwing errors because of this. Please change the model name back to `Comment`.

```suggestion
class Comment < ApplicationRecord
```